style.css

```diff
@@ -66,3 +64,54 @@ body {
   height: 342px;
   margin: 15px 0px 17px;
 }
+.fa {
+  font-size: 30px;
+  width: 30px;
+  text-align: center;
+  text-decoration: none;
+  padding: 10px;
+}
+.fa:hover {
+  background-color: var(--items-bg-color);
+  border-radius: 50%;
```

Comment:
I guess the `border-radius` effect is not only on hover, please move it to the above style for the element